chapters/stress/index.md

```diff
@@ -0,0 +1,271 @@
+# Jina Cloud Benchmark & Deployment
+
+#### Purpose: Showcase how Jina scales and how you can deploy to AWS
+
+---
+
+#### Table of Contents
+
+- [Infrastructure](#infrastructure)
+- [Datasets](#datasets)
+  * [Image search](#image-search)
+  * [Text search](#text-search)
+- [Configuration](#configuration)
+  * [Sharding and scheduling parameters](#sharding-and-scheduling-parameters)
+  * [Functional/Indexers parameter](#functional-indexers-parameter)
+  * [Client](#client)
+- [Experiments](#experiments)
+  * [Experiment 1 - Redis](#experiment-1---redis)
+    + [Results](#results)
+  * [Experiment 2 - BinaryPbIndexer](#experiment-2---binarypbindexer)
+    + [Results](#results-1)
+- [Test it yourself](#test-it-yourself)
+
+## Infrastructure
+
+This example benchmark prioritizes a real-world scenario, where a user hosts Jina in the cloud. We use six machines in AWS, of instace size `c5.2xlarge` (8 vCPUs, 16GB RAM). We also add 100GB gp2 ssd. For the Vector Indexer machine we increased RAM to 32 GB.
+
+The instances are split as follows:
+
+- **Client**: from this machine we create the Flows and issue the client requests (indexing and querying)
+- **Flow**: this machine hosts the `Gateway` to the `Flow`
+- **Encoder**: this machine processes the raw data (text or image) with the respective `Encoders`
+- **Indexer**: this machine hosts the `Indexers`
+- **Ranker**: this machine hosts the `Redis` servers and the `Rankers`
```

Comment:
Encoders, Indexers, Rankers

These are names of the Executors, right? Not the actual code we're using to refer to them. So no backticks here